packages/deploy-script-support/src/helpers.js

```diff
@@ -137,8 +137,15 @@ export const makeHelpers = async (homePromise, endowments) => {
     get getBundlerMaker() {
       return makeGetBundlerMaker(homePromise, { bundleSource, lookup });
     },
+    get writeCoreEval() {
+      return makeWriteCoreEval(homePromise, endowments, {
+        getBundleSpec: deps.cacheAndGetBundleSpec,
+        getBundlerMaker: helpers.getBundlerMaker,
+      });
+    },
+    /** @deprecated use writeCoreEval */
     get writeCoreProposal() {
```

Comment:
good to see this is is _not_ a breaking change.

We document `writeCoreProposal` externally, and I'm pretty sure it's used by 3rd parties.

https://docs.agoric.com/guides/agoric-cli/#agoric-run ->
https://github.com/Agoric/agoric-sdk/discussions/8087#discussioncomment-6534378